spec/requests/authentication_controller_spec.rb

```diff
@@ -23,12 +25,12 @@
     end
 
     it "uses a provided state_id" do
-      get sign_in_path(state_id: "hello-world")
+      get sign_in_path, params: { state_id: "hello-world" }
```

Comment:
This and L33 stand out as the only two param blocks not getting the `.to_json` treatment here.
So is the rule that we _only_ want to transform to JSON for things that are nested?

Otherwise they stay normal params?

Reply:
These are GET requests, so the params are sent in the querystring, not in the body.